views/new.ejs

```diff
@@ -9,7 +9,7 @@
         </div>
         <div class="field">
             <label>Strikes missed</label>
-            <input type="number" name="bowler[missedStrikes]" placeholder="Strikes missed">
+            <input min="0" type="number" name="bowler[missedStrikes]" placeholder="Strikes missed" required>
```

Comment:
Maybe instead of a placeholder of "Strikes missed", you could say "Initial Strike Miss Count"